### PR TITLE
Adding Dec-Jan content

### PR DESCRIPTION
**Pull request by mehtavishwa30**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,13 +105,23 @@ A maintained and comprehensive list of awesome Fuel resources!
...
+- [Sway vs. Solidity examples](https://twitter.com/schwartzswartz/status/1617902570630156294)
+- [Sway Vectors in 6 tweets or less](https://twitter.com/jtriley_eth/status/1612539679085334528)
 
 ## Libraries
 - [Sway Libraries - Fuel Labs](https://github.com/sway-libs)
 - [Swaymigo - @sproule_/](https://github.com/BlockchainCap/swaymigo)
 
 ## Developer Resources 
 - [Sway frontend template implementation](https://github.com/brech1/sway-frontend)
+- [Web3RSVP](https://github.com/camiinthisthang/learnsway-web3rsvp)
```
Comment (camiinthisthang) [inline README.md:119]: Let's not include anything until its 100% working and documented, there are still PRs open on this repo to get it working

File: README.md [hunk 2]
```diff
@@ -96,13 +105,23 @@ A maintained and comprehensive list of awesome Fuel resources!
...
+- [SwayMarketplace](https://github.com/sarahschwartz/intro-to-sway)
```
Comment (camiinthisthang) [inline README.md:120]: Link to the one on the Fuel Labs organization instead of sarah's personal

File: README.md (hunk 1 above)
Comment (mehtavishwa30) [reply README.md]: Removed this for now. Will move the repo to Fuel Labs org once it's ready and link to that fork in awesome-fuel

File: README.md (hunk 2 above)
Comment (mehtavishwa30) [reply README.md]: Done. Thanks!

**Review by camiinthisthang — APPROVED** [review]

**Pull request merged**